server/api/map_handler.py

```diff
@@ -0,0 +1,71 @@
+
+import json
+import smtplib
+import requests
+from flask import jsonify, request, Blueprint
+from config import API_KEY
+map_handler = Blueprint('map_handler', __name__)
+
+@map_handler.route('/map', methods=['POST'])
+def map():
+    if request.method != 'POST':
+        return 'NOT POST'
+    
+    postData = json.loads(request.get_data())
```

Comment:
In Python, function and variable names are in `snake_case`. Class names are in `PascalCase`. [See details](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)